app/scripts/modules/kubernetes/src/v2/manifest/edit/editManifestWizard.controller.ts

```diff
@@ -35,7 +37,10 @@ class KubernetesEditManifestCtrl implements IController {
     'ngInject';
     this.kubernetesManifestCommandBuilder.buildNewManifestCommand(application, sourceManifest, sourceMoniker)
       .then((builtCommand) => {
-        this.command = builtCommand;
+        const { command: command, metadata: metadata } = builtCommand;
```

Comment:
This can be `const { command, metadata } = builtCommand;`.

You would only use that syntax if you want a variable with a different name than the key in `builtCommand`.

Reply:
:+1: